src/main/java/org/jenkinsci/plugins/workflow/cps/CpsFlowExecution.java

```diff
@@ -1523,6 +1566,9 @@ public void marshal(Object source, HierarchicalStreamWriter w, MarshallingContex
                 for (BlockStartNode st : e.startNodes) {
                     writeChild(w, context, "start", st.getId(), String.class);
                 }
+                if (e.done != null) {
+                    writeChild(w, context, "done", e.done, Boolean.class);
```

Comment:
*glee* I can't remember off the top of my head why I care about having `done` persisted, but I know I do.